corehq/apps/smsforms/tests/test_app.py

```diff
@@ -155,3 +164,98 @@ def test_text_responses(self, tfsms_start_session_mock):
         session, responses = self._start_session(yield_responses=False)
 
         self.assertEqual(responses, ["human-error"])
+
+
+class TestFetchXml(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.session = SQLXFormsSession(session_id='abc123', domain='test-domain', user_id='user_id')
+        cls.formplayer_interface = FormplayerInterface(cls.session.session_id, cls.session.domain)
+
+    def setUp(self) -> None:
+        super().setUp()
+        patcher = patch.object(self.formplayer_interface, 'get_raw_instance')
+        self.mock_get_raw_instance = patcher.start()
+        self.addCleanup(patcher.stop)
+
+    def test_successful_response(self):
+        expected_result = '''
+        <data>
+            <question>answer</question>
+        </data>
+        '''.strip()
```

Comment:
nit: could save some lines
```suggestion
        expected_result = "<data><question>answer</question></data>"
```